gen/ad_user_mu_ucn

```diff
@@ -127,4 +151,13 @@ for my $login (@logins) {
 
 close(FILE);
 
+sub checkBase64 {
```

Comment:
I believe that all subs needs to be defined at the top like: ``sub checkBase64;`` aka - before their actual usage, but it doesn't seem to be a case even in the ``safeq`` service, from which I believe this code originates. So if it works for you, then its OK.